web/business-registry-dashboard/app/locales/fr-CA.ts

```diff
@@ -238,6 +238,11 @@ export default {
     DISSOLUTION: 'Cette entreprise est en cours de dissolution',
     PROCESSING: "Cette demande de dénomination est encore en cours de traitement, cela peut prendre jusqu'à 10 minutes.",
     EXPIRED: "Cette {type} n'est plus valide; la demande de nom est expirée.",
+    EXPIRING_SOON: {
+      today: "Cette demande de nom expirera dans Aujourd'hui",
+      tomorrow: 'Cette demande de nom expirera dans Demain',
+      inDays: 'Cette demande de nom expirera dans dans {daysDiff} jours'
```

Comment:
Should be:
   "Cette demande de nom expirera Aujourd'hui"
   "Cette demande de nom expirera Demain"
   "Cette demande de nom expirera dans {daysDiff} jours"